### DISCUSSION TITLE
HLS on proxied rtsp

### DISCUSSION THREAD

**Discussion by RichardPar** [bluenviron/mediamtx#934] [Questions]
Hi all,

Love the work! - I am puzzled how to have proxied and HLS streams at the same time. I link to RTSP CCTV cameras which works (connect to Frigate NVR) - but I cant get the HLS stream to play. 

I am connecting to port 8888 using VLC, and I can see the index and stream.m3u8 - but nothing ever plays

`paths:
  proxied_front_high:
    source: rtsp://xxxxxxx:yyyyyy@192.168.20.129:554/stream0
    sourceOnDemand: yes

  proxied_front_low:
    source: rtsp://xxxxxx:yyyyyy@192.168.20.129:554/stream1
    sourceOnDemand: yes`

(sorry) Formatting is messed up

What do I need to add to make HLS streaming work ? - The aim is to get the video to a ChromeCast device

Thanks in advance,
Richard

**Comment by aler9** [comment]
Hello, the configuration file looks nice to me, and conversion should be automatic. Please try again with the latest version, check logs on both server-side and VLC-side, and if you find any problem feel free to open an issue and provide needed data.

[a comment by an automated account is not shown]
